octoprint_pushover/__init__.py

```diff
@@ -152,6 +172,14 @@ def sent_m70(self, comm_instance, phase, cmd, cmd_type, gcode, *args, **kwargs):
 		:param kwargs: 
 		:return: 
 		"""
+		if gcode and gcode != "G1":
```

Comment:
This is so we don't calculate the time difference every `G1` which is send multiple times a second.